sdk/examples/xo_python/sawtooth_xo/processor/xo_state.py

```diff
@@ -0,0 +1,150 @@
+# Copyright 2018 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# -----------------------------------------------------------------------------
+
+import hashlib
+
+from sawtooth_sdk.processor.exceptions import InternalError
+
+
+XO_NAMESPACE = hashlib.sha512('xo'.encode("utf-8")).hexdigest()[0:6]
+
+
+def _make_xo_address(name):
+    return XO_NAMESPACE + \
+        hashlib.sha512(name.encode('utf-8')).hexdigest()[:64]
+
+
+class Game(object):
+    def __init__(self, name, board, state, player1, player2):
+        self.name = name
+        self.board = board
+        self.state = state
+        self.player1 = player1
+        self.player2 = player2
+
+
+class XoState(object):
+
+    TIMEOUT = 3
+
+    def __init__(self, context, game_name):
```

Comment:
This constructor shouldn't take a game_name, as XoState should appear to be stateless, and you should be able to access/store any number of games using this class. (Because it's a better example, not because it is used this way by the Xo game logic.)